ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -42,15 +51,29 @@ tests:
     pre:
     - ref: rosa-terraform-e2e-setup
 - as: e2e-periodic
+  cluster: build05
```

Comment:
why do we need it running in a specific cluster?

Reply:
that's the way to work via vpn to internal lab